code/game/objects/items/stacks/cable.dm

```diff
@@ -168,6 +168,51 @@ var/global/list/datum/stack_recipe/cable_recipes = list ( \
 				getFromPool(/obj/item/stack/cable_coil, C.loc)
 				returnToPool(C)
 
+//I really wish that redundant shit wasn't needed, but snowflake code yeah
+/obj/item/stack/cable_coil/proc/catwalk_place(obj/structure/catwalk/Catwalk, mob/user)
```

Comment:
There's many instructions that are redundant or outright incompatible (checking if the "turf" is intact for instance). This is a quick way to set up an exception